packages/compiler/test/cli.test.ts

```diff
@@ -34,6 +34,15 @@ describe("compiler: cli", () => {
       });
     });
 
+    it("--tspconfig-dir plumbs through", async () => {
+      const options = await resolveCompilerOptions({ "tspconfig-dir": `${cwd}/myconfigdir` });
+      deepStrictEqual(options, {
+        outputDir: `${cwd}/tsp-output`,
+        tspconfigDir: `${cwd}/myconfigdir`,
```

Comment:
Probably need to add this to cspell words

Reply:
Fixing the casing satisfied cspell :grin: